run-tests

```diff
@@ -26,13 +26,9 @@ my $luajit_prefix = shift or
 
 my $valgrind = $opts{v};
 
-my $luajit = shift;
-my $cc = shift // 'gcc';
-my $cxx = shift // 'g++';
-
-if (!defined $luajit) {
-    $luajit = glob "$luajit_prefix/bin/luajit-2.1*";
-}
+my $cc = 'gcc';
+my $cxx = 'g++';
```

Comment:
I don't like this change. This removes the original flexibility. We may want to use other C/C++ compilers like clang to run this test suite.

Reply:
Ok, I removed the commit. But we should update the `run-tests` tool somehow or fix its help message because its current documentation and behavior are mis-matched:

```
$ ./run-tests -h
Usage: ./run-tests [-h] [-v] [-j JOBS] LUAJIT-PREFIX [TEST-FILE...]
";

Options:
    -h      Print this help.
    -j N    Run tests in N parallel jobs, utilizing multiple
            CPU cores.
    -v      Use Valgrind to run the tests
$ ./run-tests /path/to/luajit2-v2.1 test/ffi/unsink_64_ptr.lua
cannot find the luajit binary under /path/to/luajit2-v2.1/bin/ at ./run-tests line 38.
```